pom.xml

```diff
@@ -44,7 +44,7 @@
         <air.maven.version>3.3.9</air.maven.version>
 
         <dep.antlr.version>4.7.1</dep.antlr.version>
-        <dep.airlift.version>0.177-ARIA-SNAPSHOT</dep.airlift.version>
+        <dep.airlift.version>0.179-masha-SNAPSHOT</dep.airlift.version>
```

Comment:
Can we use 0.179-ARIA-SNAPSHOT?

Reply:
@yingsu00 We can't because it is not available in Maven Central and we don't have permissions to publish there.